lib/ts/recipe/accountlinking/types.ts

```diff
@@ -65,13 +64,38 @@ export type TypeNormalisedInput = {
             originalImplementation: RecipeInterface,
             builder?: OverrideableBuilder<RecipeInterface>
         ) => RecipeInterface;
-        apis: (originalImplementation: APIInterface, builder?: OverrideableBuilder<APIInterface>) => APIInterface;
     };
 };
 
-export type APIInterface = {};
-
 export type RecipeInterface = {
+    getRecipeUserIdsForPrimaryUserIds: (input: {
+        primaryUserIds: string[];
+        userContext: any;
+    }) => Promise<{
+        [primaryUserId: string]: string[]; // recipeUserIds
+    }>;
+    getPrimaryUserIdsforRecipeUserIds: (input: {
+        recipeUserIds: string[];
+        userContext: any;
+    }) => Promise<{
+        [recipeUserId: string]: string | null;
```

Comment:
```suggestion
        [recipeUserId: string]: string | null; // if recipeUserId doesn't have a primaryUserId, then it will be mapped to `null`. If the input recipeUserId doesn't exist, then it won't be a part of the map
```